dotCMS/src/main/java/com/dotcms/security/apps/AppsAPIImpl.java

```diff
@@ -1005,4 +1009,146 @@ public void resetSecrets(final User user)
        appsCache.clearCache();
     }
 
+    /**
+     *
+     * @param key
+     * @param paramAppKeysBySite
+     * @return
+     */
+    public File exportSecrets(final Key key, final boolean exportAll,
+            final Map<String, Set<String>> paramAppKeysBySite, final User user)
+            throws DotDataException, DotSecurityException, IOException {
+
+        if(!userAPI.isCMSAdmin(user)){
+            throw new DotSecurityException("Only Admins are allowed to perform an export operation.");
+        }
+
+        final AppsSecretsImportExport exportedSecrets;
```

Comment:
final AppsSecretsImportExport exportedSecrets =
       exportAll?collectSecretsForExport(appKeysByHost(), user): collectSecretsForExport(paramAppKeysBySite, user);

Reply:
in my opinion not everything needs to be converted to ternary operations only because we can. its a matter of style and prefference